server_environment.yml

```diff
@@ -8,7 +8,6 @@ dependencies:
 - mesalib
 - ffmpeg
 - vtk
-- traits
```

Comment:
Did you test this without migrating to GH actions at all? This might already be enough to fix it actually.

But I'm fine with moving to GH actions, so I'll merge anyway!

Reply:
I removed it in https://github.com/mne-tools/mne-python/pull/10378 and it was green (52 minutes instead of timeout) but one commit later it was timeout again.

https://github.com/mne-tools/mne-python/runs/5408438264